integration-tests/docker-compose.test.status-go.yml

```diff
@@ -21,6 +21,7 @@ services:
       retries: 120
 
   tests-rpc:
+    user: ${INTEGRATION_TESTS_DOCKER_UID}
```

Comment:
Does this work when it's empty?

Reply:
Yes, I think so. Initially it was `$UID`, which was empty